src/rdf4cpp/rdf/IRIFactory.cpp

```diff
@@ -0,0 +1,360 @@
+#include "IRIFactory.hpp"
```

Comment:
Please split IRIFactory and IRIView into seperate files, just to make them shorter